### DISCUSSION TITLE
Forge Port

### DISCUSSION THREAD

**Discussion by Chancho91** [Draylar/identity#261] [General]
A Forge port of this mod would be great. Please consider!

**Comment by James103** [comment]
> A Forge port of this mod would be great. Please consider!

The latest versions of [Metamorph](https://github.com/mchorse/metamorph) and [Changeling](https://github.com/asanetargoss/Changeling) are for Forge, but are only for Minecraft versions up to 1.12.2, whereas Identity is on 1.16.5. This means there is likely a valid gap that could be filled by porting Identity to Forge 1.16.5 or 1.17+, should the developer should choose.

**Comment by Draylar** [comment]
![image](https://user-images.githubusercontent.com/17728338/139620817-90de8108-1119-485c-b569-b06c9aef1a9c.png)

😉
